styles/new/base.css

```diff
@@ -0,0 +1,15 @@
+* {
+  @apply subpixel-antialiased;
+}
+
+html,
+body {
```

Comment:
we should add `scroll-smooth` to allow smooth scrolling when using ToC

Reply:
Why?

Reply:
when you click on TOC it's popping on the section with smooth scroll it's will be smooth

Reply:
I understand that, but why we want smooth scrolling? Just for the effect?

Reply:
> Just for the effect?

yes and no. Yes because it's an effect, but it's also for this long page. Because when you click, you scroll up or down, which helps you keep in mind where you are in the article.